test/jdk/java/awt/TrayIcon/ActionEventTest/ActionEventTest.java

```diff
@@ -26,16 +26,12 @@
  * @bug 6191390 8154328
  * @key headful
  * @summary Verify that ActionEvent is received with correct modifiers set.
- * @modules java.desktop/java.awt:open
- * @modules java.desktop/java.awt.peer
- * @library /lib/client ../
- * @library /java/awt/patchlib
- * @build java.desktop/java.awt.Helper
- * @build ExtendedRobot SystemTrayIconHelper
+ * @modules java.desktop/java.awt:open java.desktop/java.awt.peer
+ * @library /lib/client ../ /java/awt/patchlib
```

Comment:
Is anything else but `ExtendedRobot` used from `/lib/client`? If not, remove it.